rules-reviewed/eap7/eap71/hibernate51-53.windup.xml

```diff
@@ -224,17 +224,24 @@
                 </hint>
             </perform>
         </rule>
-        <!--https://issues.jboss.org/browse/WINDUPRULE-381
+        <!--https://issues.jboss.org/browse/WINDUPRULE-381 -->
         <rule id="hibernate51-53-00500">
             <when>
+                <javaclass references="org.hibernate.{className}" >
+                </javaclass>
             </when>
             <perform>
+                <hint title="Hibernate 5.3 - RegionFactory usage" effort="2" category-id="mandatory">
+                    <message>Review usage of classes in org.hibernate.cache.spi.RegionFactory, as SPIs for this class has changed. See Javadoc for org.hibernate.cache.spi.RegionFactory for details.</message>
```

Comment:
with 2 spaces and a new line it will be rendered as a new line in markdown
```suggestion
                    <message>Review usage of classes in org.hibernate.cache.spi.RegionFactory, as SPIs for this class has changed.  
                    See Javadoc for org.hibernate.cache.spi.RegionFactory for details.</message>
```